recipes/xtensor/all/conanfile.py

```diff
@@ -103,6 +106,8 @@ def package_info(self):
         self.cpp_info.build_modules["cmake_find_package"] = [self._module_file_rel_path]
         self.cpp_info.build_modules["cmake_find_package_multi"] = [self._module_file_rel_path]
         self.cpp_info.names["pkg_config"] = "xtensor"
+        if self.settings.compiler in ("gcc", "clang", "apple-clang"):
+            self.cpp_info.defines.append("_GLIBCXX_USE_CXX11_ABI=1")
```

Comment:
I think it should be added conditionally on `compiler.libcxx` value

Reply:
I agree with you.
I will try to fix later.

Reply:
Why you need to force it? The consumer should enable C++14 need instead.

Reply:
I think xtensor checks _GLIBCXX_USE_CXX11_ABI definition.
[documentation](https://github.com/xtensor-stack/xtensor/blob/78aaac39143caa78da7c5c0734ccef957535f0c0/docs/source/compilers.rst#gcc--51-and-stdis_trivially_default_constructible)
[source code which uses _GLIBCXX_USE_CXX11_ABI](https://github.com/xtensor-stack/xtensor/blob/d5a5c6332b72d249f3a1495e1eaf447f5e1e9d28/include/xtensor/xutils.hpp#L640)

Reply:
conan automatically defined `_GLIBCXX_USE_CXX11_ABI` to an appropriate value, depending on `self.settings.compiler.libcxx`:
-  `libstdc++` will do `_GLIBCXX_USE_CXX11_ABI=0`
-  `libstdc++11` will do `_GLIBCXX_USE_CXX11_ABI=1`

These lines MUST be removed, because this will result in usage of `libstdc++11`, even when `self.settings.compiler.libcxx == "libstdc++'`.

Reply:
Now I see. I will remove it.
Also I will try to find a way to fix compilation error without _GLIBCXX_USE_CXX11_ABI=1.